Panic on bad configuration file

Using a bad configuration file, Traefik panics.
```toml
[frontends]
  [frontends.frontend1]
  backend = "backend1"
    [frontends.frontend1.routes.test_1]
    rule = "Host"
    value = "rlon6667-cca.intra.laposte.fr"
    [frontends.frontend1.routes.test_2]
    rule = "Path"
    value = "/"
    [frontends.frontend1.routes.test_3]
    rule = "passHostHeader"
    value = "true"
```
```
[root@RLON6666 traefik]# ./traefik traefik.toml
panic: reflect: call of reflect.Value.Call on zero Value

goroutine 9 [running]:
reflect.flag.mustBe(0x0, 0x13)
        /usr/local/go/src/reflect/value.go:199 +0x8b
reflect.Value.Call(0x0, 0x0, 0x0, 0x18c99760, 0x1, 0x1, 0x0, 0x0, 0x0)
        /usr/local/go/src/reflect/value.go:298 +0x3b
main.Invoke(0x8593f40, 0x18c9a900, 0x18c99350, 0x10, 0x18c5fcd0, 0x1, 0x1, 0x0, 0x0, 0x0)
        /go/src/github.com/emilevauge/traefik/traefik.go:362 +0x1ac
main.LoadConfig(0x18cb6020, 0x18b6f4a0, 0x18c5ff1c, 0x0, 0x0)
        /go/src/github.com/emilevauge/traefik/traefik.go:293 +0x88e
main.main.func2(0x18b93880, 0x18b6f4a0, 0x18b24c58, 0x18b24c50, 0x18b25a50)
        /go/src/github.com/emilevauge/traefik/traefik.go:125 +0x441
created by main.main
        /go/src/github.com/emilevauge/traefik/traefik.go:146 +0x967

goroutine 1 [chan receive]:
main.main()
        /go/src/github.com/emilevauge/traefik/traefik.go:208 +0x132d

goroutine 5 [syscall]:
os/signal.loop()
        /usr/local/go/src/os/signal/signal_unix.go:22 +0x1a
created by os/signal.init.1
        /usr/local/go/src/os/signal/signal_unix.go:28 +0x36

goroutine 6 [sleep]:
time.Sleep(0x3b9aca00, 0x0)
        /usr/local/go/src/runtime/time.go:59 +0xe6
github.com/emilevauge/traefik/vendor/github.com/thoas/stats.New.func1(0x18b185f0)
        /go/src/github.com/emilevauge/traefik/vendor/github.com/thoas/stats/stats.go:33 +0x35
created by github.com/emilevauge/traefik/vendor/github.com/thoas/stats.New
        /go/src/github.com/emilevauge/traefik/vendor/github.com/thoas/stats/stats.go:35 +0x152

goroutine 7 [select, locked to thread]:
runtime.gopark(0x86af1f0, 0x18b22f8c, 0x85aa3b0, 0x6, 0x8934f18, 0x2)
        /usr/local/go/src/runtime/proc.go:185 +0x12d
runtime.selectgoImpl(0x18b22f8c, 0x0, 0xc)
        /usr/local/go/src/runtime/select.go:392 +0x9ee
runtime.selectgo(0x18b22f8c)
        /usr/local/go/src/runtime/select.go:212 +0xf
runtime.ensureSigM.func1()
        /usr/local/go/src/runtime/signal1_unix.go:227 +0x2f9
runtime.goexit()
        /usr/local/go/src/runtime/asm_386.s:1662 +0x1

goroutine 8 [chan receive]:
main.main.func1(0x18b93840, 0x18b6f4a0, 0x18b93880)
        /go/src/github.com/emilevauge/traefik/traefik.go:90 +0x1aa
created by main.main
        /go/src/github.com/emilevauge/traefik/traefik.go:109 +0x924

goroutine 12 [chan receive]:
main.main.func4(0x18b938c0, 0x18b24c50, 0x18b93900)
        /go/src/github.com/emilevauge/traefik/traefik.go:191 +0x3f
created by main.main
        /go/src/github.com/emilevauge/traefik/traefik.go:196 +0x1071
```

--- a/traefik.go
+++ b/traefik.go
@@ -290,7 +290,10 @@ func LoadConfig(configurations configs, globalConfiguration *GlobalConfiguration
 			newRoute := router.NewRoute().Name(frontendName)
 			for routeName, route := range frontend.Routes {
 				log.Debugf("Creating route %s %s:%s", routeName, route.Rule, route.Value)
-				newRouteReflect := Invoke(newRoute, route.Rule, route.Value)
+				newRouteReflect, err := invoke(newRoute, route.Rule, route.Value)
+				if err != nil {
+					return nil, err
+				}
 				newRoute = newRouteReflect[0].Interface().(*mux.Route)
 			}
 			if backends[frontend.Backend] == nil {
@@ -354,10 +357,14 @@ func LoadConfig(configurations configs, globalConfiguration *GlobalConfiguration
 
 // Invoke calls the specified method with the specified arguments on the specified interface.
 // It uses the go(lang) reflect package.
-func Invoke(any interface{}, name string, args ...interface{}) []reflect.Value {
+func invoke(any interface{}, name string, args ...interface{}) ([]reflect.Value, error) {
 	inputs := make([]reflect.Value, len(args))
 	for i := range args {
 		inputs[i] = reflect.ValueOf(args[i])
 	}
-	return reflect.ValueOf(any).MethodByName(name).Call(inputs)
+	method := reflect.ValueOf(any).MethodByName(name)
+	if method.IsValid() {
+		return method.Call(inputs), nil
+	}
+	return nil, errors.New("Method not found: " + name)
 }

